src/number_format.js

```diff
@@ -24,7 +25,8 @@ const propTypes = {
 
 const defaultProps = {
   displayType: 'input',
-  decimalSeparator: '.'
+  decimalSeparator: '.',
+  decimalPrecision: false
```

Comment:
Not sure I understand your comment. If they pass false to the decimalPrecision, it would be unlimited number of decimal places. Could you explain this a little more?